benchmark.py

```diff
@@ -8,6 +8,7 @@
 import time
 
 import pexpect
+import python_helpers as python_utils
```

Comment:
I would just call this `utils` at the filename and Python level, it's a bit tautological to include Python in a Python module name.